src/Data/Formatter/Number.purs

```diff
@@ -41,8 +43,18 @@ newtype Formatter = Formatter
   , after :: Int
   , abbreviations :: Boolean
   , sign :: Boolean
+  , groupSeparator :: Char
+  , decimalSeparator :: Char
   }
 
+-- | change the default Separators
+-- | for example for german formatting you could do
+-- |
+-- | > parseFormatString ".." # map (changeSeparators { groupSeparator: '.', decimalSeparator: ','})
+changeSeparators :: { groupSeparator :: Char, decimalSeparator :: Char } -> Formatter -> Formatter
+changeSeparators { groupSeparator, decimalSeparator } (Formatter formatter) =
```

Comment:
I'd suggest calling this `withSeparators` perhaps? But other than that this looks good to me!

Reply:
no problem - guess it sounds better on use-side this way - I'll change it